templates/gcp/fips.html

```diff
@@ -0,0 +1,365 @@
+{% extends "azure/base_azure.html" %}
+
+{% block title %}Ubuntu Pro FIPS for GCP{% endblock %}
+{% block meta_description %}Ubuntu Pro FIPS for Google Cloud is an Ubuntu Pro image with FIPS certified modules enabled by default for FedRAMP, HIPAA, PCI and ISO compliance. Ubuntu Pro FIPS images are secure and optimised for production on public cloud.{% endblock %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1Djxdzjy78bau8ELM4-MNGoM2YRa_QjpN1xJ4H4GjFyw/edit#heading=h.c6wjbvvp8mcm{% endblock meta_copydoc %}
+
+{% block content %}
+<section class="p-strip u-no-padding--top u-no-padding--bottom">
+  <div class="p-strip--suru-topped">
+    <div class="row u-equal-height">
+      <div class="col-7 col-medium-4">
+        <h1>Ubuntu Pro FIPS for Google Cloud compliance</h1>
+        <h3>For Google Cloud compliance in production environments.</h3>
```

Comment:
Accessibility practices dictate that headings most follow hierarchy, so only a h2 can can below a h1.

Reply:
mm ok but you guys usually tell me to not do `<h2 class="p-heading--3"`?